phys2bids/tests/test_integration.py

```diff
@@ -5,11 +5,12 @@
 import shutil
 import subprocess
 from os import remove
-from os.path import isfile, join, split
+from os.path import isfile, join, split, dirname
 from pkg_resources import resource_filename
 
 import pytest
 
+import phys2bids as p2b
```

Comment:
Although I like it, I'm not sure this is the best way to go - can you reorder a little the phys2bids imports?

Thinking about it, shouldn't it be:
```
import .
from ._version import get_versions
```
etc.etc.?

Reply:
I know it's not the nicest way of doing things but I had to because I need to access the following attribute: `p2b.__file__`.

Given that we are calling the package and the main function `phys2bids`, there's a conflict there. 

If you have a better idea of how we could deal with that, I'm all ears.

Reply:
`from phys2bids import __file__`?

Also please move from pkg_resources before phys2bids (being third party non-local)